src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -313,17 +313,9 @@ public async Task PutResetPasswordEnrollment(string orgId, string userId, [FromB
             throw new UnauthorizedAccessException();
         }
 
-        if (model.ResetPasswordKey != null && !await _userService.VerifySecretAsync(user, model.Secret))
-        {
-            await Task.Delay(2000);
-            throw new BadRequestException("MasterPasswordHash", "Invalid password.");
-        }
-        else
-        {
-            var callingUserId = user.Id;
-            await _organizationService.UpdateUserResetPasswordEnrollmentAsync(
-               new Guid(orgId), new Guid(userId), model.ResetPasswordKey, callingUserId);
-        }
+        var callingUserId = user.Id;
+        await _organizationService.UpdateUserResetPasswordEnrollmentAsync(
+            new Guid(orgId), new Guid(userId), model.ResetPasswordKey, _userService, callingUserId);
```

Comment:
❓ : Should this change be feature flagged behind TDE?

Reply:
Hmm, good point, I'd personally don't think so but I'll check!

Reply:
Resolving as slack conversation has indicated that a feature flag is not necessary in this context. Added complexity for no real win.